src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -97,6 +104,13 @@ protected function createConfigurationDefinition()
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder(
+                'ensure_fully_multiline',
+                'Ensure every argument of a multiline argument lists is on its own line'
```

Comment:
You could also write "each and every", so I guess this means both are probably equally valid